numpy/linalg/lapack_lite/clapack_scrub.py

```diff
@@ -2,11 +2,17 @@
 from __future__ import division, absolute_import, print_function
 
 import sys, os
-from io import BytesIO
 import re
 from plex import Scanner, Str, Lexicon, Opt, Bol, State, AnyChar, TEXT, IGNORE
 from plex.traditional import re as Re
 
+PY2 = sys.version_info < (3, 0)
+
+if PY2:
+    from io import BytesIO
+else:
+    from io import StringIO as BytesIO
```

Comment:
This looks pretty sketchy - perhaps call it `UStringIO` which would at least match the internal name we use for the python 2/3 string type, or perhaps `TextIO` which I think we use in tests somewhere.

Reply:
Okay.